src/components/component-image/ComponentImage.tsx

```diff
@@ -1,8 +1,8 @@
-import React, { ReactNode } from 'react';
-import { SComponentImage } from './styles/SComponentImage';
-import { SInnerRectangle } from './styles/SInnerRectangle';
+import React, { ReactNode } from "react";
+import { SComponentImage } from "./styles/SComponentImage";
+import { SInnerRectangle } from "./styles/SInnerRectangle";
 
-export type IComponentImageTypes = "open" | "grid" | "list item";
+export type IComponentImageTypes = "open" | "grid" | "list-item";
```

Comment:
@nicolasshiken you have changed this type but not the story.

Reply:
The story used `list-item` already, and the styledComponents use `list-item` as well, that's why I changed the interface. Unless I misunderstood?

Reply:
The story uses `list-item` already, and the styledComponents use `list-item` as well, that's why I changed the type. Or did I misunderstood something?